gcloud-java-dns/src/test/java/com/google/gcloud/dns/DnsRecordTest.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.gcloud.dns;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import static org.junit.Assert.assertNotEquals;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import org.easymock.EasyMock;
+
+
+public class DnsRecordTest {
+
+  private static final String NAME = "example.com.";
+  private static final Integer TTL = 3600;
+  private static final DnsRecord.DnsRecordType TYPE = DnsRecord.DnsRecordType.AAAA;
+  private static final ManagedZoneInfo MANAGED_ZONE_INFO_MOCK =
+          EasyMock.createMock(ManagedZoneInfo.class);
+  private static final Long ZONE_ID = 12L;
+  private static final String ZONE_NAME = "name";
+
+  static {
```

Comment:
My preference is for moving this code (and `RECORD`) initialization to a static `@BeforeClass` method even though that requires us to remove `final`. I think it's nice to have mock creation inside a method rather than in a static block (for readability/maintainability and consistency) if the only prize to pay is removing `final` to a variable used in tests.